subsys/bluetooth/common/dummy.c

```diff
@@ -10,6 +10,17 @@
  */
 
 #include <zephyr.h>
+#include <bluetooth/bluetooth.h>
+
+/*
+ * The unpacked structs below are used inside __packed structures that reflect
+ * what goes on the wire. While alignment is not a problem for those, since all
+ * their members are bytes or byte arrays, the size is. They must not be padded
+ * by the compiler, otherwise the on-wire packet will not map the packed
+ * structure correctly.
+ */
+BUILD_ASSERT(sizeof(bt_addr_t) == 6);
+BUILD_ASSERT(sizeof(bt_addr_le_t) == 7);
```

Comment:
If it is just for these two, the commit message could have been more explicit, but it's OK.

Reply:
Sure, I can fix that, no worries.

Reply:
Define the following in `include/bluetooth/addr.h` 

`#define BT_ADDR_SIZE 6`

and 

`#define BT_ADDR_LE_SIZE 7`

close to respective struct definitions. Then,


```suggestion
BUILD_ASSERT(sizeof(bt_addr_t) == BT_ADDR_SIZE);
BUILD_ASSERT(sizeof(bt_addr_le_t) == BT_ADDR_LE_SIZE);
```